main.cpp

```diff
@@ -52,85 +52,90 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: angryDog
+//  action 1: the dog chases the postman
+angryDog.chasePostman();
+//  action 2: the dog barks
+angryDog.bark();
+//  action 3: the dog chews the slippers
+angryDark.chewSlippers();
 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: trafficLights
+//  action 1: the traffic lights turn red
+trafficLights.turnRed();
+//  action 2: the traffic lights turn green
+trafficLights.turnGreen();
+//  action 3: the traffic lights break
+trafficLights.break();
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: shark
+//  action 1: the shark swims
+shark.swim();
+//  action 2: the shark eats a fish
+shark.eatFish();
+//  action 3: the shark attacks the boat
+shark.attackBoat();
+
+//  4) 
+//  Noun: cyclist
+//  action 1: The cyclist rides their bike
+cyclist.rideBike();
+//  action 2: The cyclist pedals faster.
+cyclist.pedalFast();
+//  action 3: The cyclist falls off the bike
+cylist.fallOffBike();
 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: flea
+//  action 1: the flea hops
+flea.hop();
+//  action 2: the flea bites the dog
+flea.biteDog();
+//  action 3: the flea dies
+flea.die();
+
+//  6) 
+//  Noun: mouth
+//  action 1: the mouth talks
+mouth.talk();
+//  action 2: the mouth chews
+mouth.chew();
+//  action 3: the mouth smiles
+mouth.smile();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: clock
+//  action 1: the clock ticks
+clock.tick();
+//  action 2: the clock tocks
+clock.tock();
+//  action 3: the clock chimes
+clock.chime();
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: tree
+//  action 1: the tree grows
+tree.grow();
+//  action 2: the tree sways
+tree.sway();
+//  action 3: the tree falls over
+tree.fallOver();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.